charts/shoot-core/charts/csi-alicloud/templates/csi-attacher-dep.yaml

```diff
@@ -0,0 +1,61 @@
+{{- if .Values.enabled }}
+kind: Deployment
+apiVersion: {{ include "deploymentversion" . }}
+metadata:
+  name: csi-attacher
```

Comment:
It would be better to add a suffix to all those CSI deployments/daemonsets and secrets similar to 

https://github.com/kubernetes/cloud-provider-openstack/blob/f056677572b2635632abcc7dbde459cdfc4432b9/manifests/cinder-csi-plugin/csi-nodeplugin-cinderplugin.yaml#L7 

End-users might also install their own CSI plugins and it might be better to more easily distinguish between them.